examples/mysql-galera/pxc-node3-controller.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    name: pxc-node3 
+  name: pxc-node3 
+spec:
+  replicas: 1
+  selector:
+    name: pxc-node3 
```

Comment:
[best practices](https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/config-best-practices.md)#9: Don't specify default values.

In the case of replication controllers, the controller's selector defaults to the template's labels.